backend/src/test/java/com/github/jekattack/cornergame/game/quests/QuestServiceTest.java

```diff
@@ -0,0 +1,113 @@
+package com.github.jekattack.cornergame.game.quests;
+
+import com.github.jekattack.cornergame.game.gamedata.CGUserGameData;
+import com.github.jekattack.cornergame.game.gamedata.CGUserGameDataRespository;
+import com.github.jekattack.cornergame.game.gamedata.CGUserGameDataService;
+import com.github.jekattack.cornergame.game.gamedata.questItem.QuestItem;
+import com.github.jekattack.cornergame.game.gamedata.questItem.QuestStatus;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.time.Instant;
+import java.util.*;
+
+class QuestServiceTest {
+
```

Comment:
some edge cases are left here i think, fail to create, maybe adding something double...